app/models/location.rb

```diff
@@ -46,7 +46,9 @@ def self.location_api_request(endpoint_query)
     resp = Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
       http.request(request)
     end
-    [resp.is_a?(Net::HTTPSuccess), JSON.parse(resp.body)]
+    # Search with 0 results will return HTTPNotFound. Consider it a successful request for our handling.
+    success = resp.is_a?(Net::HTTPSuccess) || resp.is_a?(Net::HTTPNotFound)
```

Comment:
Could HTTPNotFound be an error on our part ? Or if for some reason something changes on our config?

Reply:
sounds like it's possible yeah. what would you suggest for this?

Reply:
So, with your changes above any other errors will be logged, right?

Reply:
right, they will fall through and go to log and Airbrake